Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Validation/BocListRowAndCellValidationFailureHandler.cs

```diff
@@ -0,0 +1,61 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// using NUnit.Framework;
+//
+using System;
+using System.Linq;
+using Remotion.Globalization;
+using Remotion.ServiceLocation;
```

Comment:
```suggestion
using Remotion.ObjectBinding.Web.UI.Controls.BocListImplementation;
using Remotion.ServiceLocation;
```

Reply:
This isn't required anymore as we moved these classes.

Reply:
ah, right. yes, parent namespace.